src/_nebari/stages/kubernetes_services/template/modules/kubernetes/services/conda-store/worker.tf

```diff
@@ -202,3 +206,71 @@ resource "kubernetes_deployment" "worker" {
     }
   }
 }
+
+resource "kubernetes_manifest" "triggerauthenticator" {
+  manifest = {
+    apiVersion = "keda.sh/v1alpha1"
+    kind       = "TriggerAuthentication"
+
+    metadata = {
+      name      = "trigger-auth-postgres"
+      namespace = var.namespace
+    }
+
+    spec = {
+      secretTargetRef = [
+        {
+          name      = "nebari-conda-store-postgresql"
+          parameter = "password"
+          key       = "postgresql-password"
+        }
+      ]
+    }
+  }
+  depends_on = [
+    kubernetes_deployment.worker
+  ]
+}
+
+resource "kubernetes_manifest" "scaledobject" {
+  manifest = {
+    apiVersion = "keda.sh/v1alpha1"
+    kind       = "ScaledObject"
+
+    metadata = {
+      name      = "scaled-conda-worker"
+      namespace = var.namespace
+    }
+
+    spec = {
+      scaleTargetRef = {
+        kind = "Deployment"
+        name = "nebari-conda-store-worker"
+      }
+      maxReplicaCount = var.max-worker-replica-count
+      pollingInterval = 5
+      cooldownPeriod  = 5
+      triggers = [
+        {
+          type = "postgresql"
```

Comment:
I wouldn't necessarily prevent merging b/c of this, but this would be more robust if we use the conda store api instead of querying the database directly (since the db design could change).  I haven't seen a way to run a curl or custom command for a trigger.  Are you familiar with any way?

Reply:
The metrics are exported to Prometheus so we could use the Prometheus trigger, but users can disable monitoring and we'd still want the scaling to work so not ideal.  I also see they have custom [external scalers]( https://keda.sh/docs/2.14/concepts/external-scalers/), but that seems like a pain to implement when this works as you have it.

Reply:
Your comment about SQL got me thinking ... 
You are correct that SQL is not an exposed AP and, therefore, has no backward compatibility guarantees.
I found the solution in KEDA documentation about  [Metric API scaler](https://keda.sh/docs/2.12/scalers/metrics-api/).
<img width="733" alt="keda-metric-api" src="https://github.com/nebari-dev/nebari/assets/8033215/9e94c9b3-521e-4730-99c0-27f43c38a30c">

So we need to 
1. give Keda credentials to hit
`https://nebari.quansight.dev/conda-store/api/v1/build/?status=QUEUED`
and 
`https://nebari.quansight.dev/conda-store/api/v1/build/?status=BUILDING`
2. and, modify this ScaledObject or to have 2 triggers one for each endpoint. 

Now, I just need to figure out how to grant permission to KEDA operator to access conda-store API.